packages/terra-pills/src/private/Pill.module.scss

```diff
@@ -0,0 +1,430 @@
+@import '~terra-mixins/lib/Mixins';
+
+// Themes
+@import './../clinical-lowlight-theme/Pill.module';
+@import './../orion-fusion-theme/Pill.module';
+/* stylelint-disable selector-max-compound-selectors */
+/* stylelint-disable max-nesting-depth */
```

Comment:
I'm not necessarily against disabling these stylelint rules because I know they can be next to impossible to meet on some of these more complex components, I just want to make sure that attempts to simplify were made. I looks like there are several places in this file where there are nested selectors where it's unnecessary such as:
https://github.com/cerner/terra-framework/pull/1490/files#diff-785e07d9893a6a598e72488ab91a24f6c46e27d6e2c2da8ac97b4456e66f02c6R54
https://github.com/cerner/terra-framework/pull/1490/files#diff-785e07d9893a6a598e72488ab91a24f6c46e27d6e2c2da8ac97b4456e66f02c6R99
https://github.com/cerner/terra-framework/pull/1490/files#diff-785e07d9893a6a598e72488ab91a24f6c46e27d6e2c2da8ac97b4456e66f02c6R190
https://github.com/cerner/terra-framework/pull/1490/files#diff-785e07d9893a6a598e72488ab91a24f6c46e27d6e2c2da8ac97b4456e66f02c6R286

Reply:
These have been resolved.